magento/adobe-stock-integration
# #912: Fixed image located in the media gallery

### Description (*)

This PR fixes the image located in the media gallery.

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#912: Locate Adobe Stock grid functionality has to be adjusted to work with both media galleries

### Manual testing scenarios (*)
1. Login to Admin panel
2. Open Media Gallery (i.e. Catalog -> Category -> expand Content -> Select from Gallery button)
3. Click "Search Adobe Stock"
4. Open preview for an image that was saved from the Adobe Stock previously
5. Click `Open in Media Gallery` button (this image should be selected in the media gallery)
